android/src/main/java/io/invertase/googlemobileads/ReactNativeGoogleMobileAdsBannerAdViewManager.java

```diff
@@ -123,7 +126,7 @@ public void setSizes(ReactViewGroup reactViewGroup, ReadableArray value) {
       }
     }
 
-    if (sizeList.size() > 0) {
+    if (sizeList.size() > 0 && !sizeList.contains(AdSize.FLUID)) {
```

Comment:
No need to send dimensions to RN as RN View is set as `flex: 1` and the native view can now layout itself